packages/editor/src/pages/Editor/store/settings/utilities.ts

```diff
@@ -72,34 +72,34 @@ const getSettingsFiles = (
   timestamp: number,
   userSettings: Partial<ISettings>,
 ): IFile[] => [
-  {
-    id: USER_SETTINGS_FILE_ID,
-    name: 'User Settings',
-    dateCreated: timestamp,
-    dateLastModified: timestamp,
-    dateLastOpened: timestamp,
-    language: SETTINGS_JSON_LANGUAGE,
-    content: getUserSettingsContent(userSettings),
-  },
-  {
-    id: DEFAULT_SETTINGS_FILE_ID,
-    name: 'Default Settings',
-    dateCreated: timestamp,
-    dateLastModified: timestamp,
-    dateLastOpened: timestamp,
-    language: SETTINGS_JSON_LANGUAGE,
-    content: getDefaultSettingsContent(userSettings),
-  },
-  {
-    id: ABOUT_FILE_ID,
-    name: 'About',
-    dateCreated: timestamp,
-    dateLastModified: timestamp,
-    dateLastOpened: timestamp,
-    language: 'plaintext',
-    content: getAboutContent(),
-  },
-];
+    {
```

Comment:
The pipeline is failing due to these code formatting issues. Run `npm run prettier:write` to fix them

Reply:
thanks! a new PR has sent by run this command